message_definitions/v1.0/ras_a.xml

```diff
@@ -174,6 +174,16 @@
         <param index="2" label="X Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in X direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
         <param index="3" label="Y Position in the camera frame" minValue="0" increment="1">Position given by the number of pixels in Y direction of the camera frame. Origin on the bottom left corner of the camera frame.</param>
       </entry>
+      <entry value="41051" name="MAV_CMD_DO_FOLLOW_POI" hasLocation="false" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>Invoke platform to begin following a POI as a subject</description>
+        <param index="1" label="POI UID" minValue="0" maxValue="4294967295" increment="1">UID of the POI to follow</param>
```

Comment:
Make it type safe.

Reply:
Hi @TSC21 - by making it type-safe, do you mean limiting the `maxValue` to `2,147,483,647` (i.e., the largest integer that can be safely represented as a float)?

Reply:
Yes correct